Documentation/Storage-Configuration/NFS/nfs-security.md

```diff
@@ -35,11 +37,14 @@ system (e.g., LDAP). The file follows the `sssd.conf` format documented in its
 Methods of providing the configuration file are documented in the
 [NFS CRD security section](../../CRDs/ceph-nfs-crd.md#security).
 
-The SSSD configuration file may be omitted from the CephNFS spec if desired. In this case, Rook will
-not set the `/etc/sssd/sssd.conf` in any way. This allows you to manage the `sssd.conf` file
-yourself however you wish. For example, you may build it into your custom Ceph container image, or
-use the [Vault agent injector](https://www.vaultproject.io/docs/platform/k8s/injector) to securely
-add the file via annotations on the CephNFS spec (passed to the NFS server pods).
+Notes:
+- The SSSD sidecar only requires the namespace switch (a.k.a. "nsswitch" or "nss"). We recommend
```

Comment:
```suggestion
- The SSSD sidecar only requires the namespace switch (a.k.a. `nsswitch` or `nss`). We recommend
```
?

Reply:
nss is called out as `nss` on the next line. These aren't meant to be option or tool names that should be in code syntax.